### PR TITLE
Refactoring/fix timer functionality and form validation

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

Disabled alert, user can't click timer-button, while timer runs

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by tylrs** [comment]
I bet we could update some of the button styling and spacing on the timer

File: styles.css [hunk 1]
```diff
@@ -34,7 +34,7 @@ form {
   padding-bottom: 3%;
   background-color: #46424D;
   justify-content: space-around;
-  height: 100%;
```
Comment (hoomberto) [inline styles.css:37]: This makes sense to reduce the height property's value here

File: main.js [hunk 2]
```diff
@@ -119,6 +120,7 @@ function validateIcons() {
       activeCount++
       if (activeCount === 3) {
       addError(0);
+      break;
```
Comment (hoomberto) [inline main.js:131]: Can't believe this tiny break makes it all work!

File: main.js [hunk 3]
```diff
@@ -50,6 +50,7 @@ function renderCurrentActivity() {
 
 function startCountDown() {
   countDown(currentActivity.minutes, currentActivity.seconds);
+  startTimerButton.disabled = true;
```
Comment (hoomberto) [inline main.js:55]: Setting the timer up to be disabled after click is ingenius!

**Pull request merged**